Datasets/data-connect-dataset-onedrivesynchealth.md

```diff
@@ -0,0 +1,140 @@
+| TotalErrorCount | int | Total count of all errors the user is experiencing | No | False |
+| ErrorDetails | array | Error details for this device | No | False |
+| ErrorDetails, OneDriveDeviceId | string | Unique OneDrive Sync internal identifier | No | False |
+| ErrorDetails, SyncErrorId | int | Error type, when shown in system tray icon (34 means unused) | No | False |
+| ErrorDetails, TaskDialogId1 | int | Error type, when shown in task dialog (0 means unused) | No | False |
+| ErrorDetails, TaskDialogId2 | int | Error type, when shown in task dialog, legacy (256 means unused) | No | False |
+| ErrorDetails, ErrorTitle | string | Error title shown to user (localized) | No | False |
+| ErrorDetails, ErrorDescription | string | Error description shown to user (localized) | No | False |
+| ErrorDetails, ErrorCount | int | Number of errors of this type | No | False |
+| SnapshotDate | datetime | When this site information was captured (in UTC) | Yes | True |
+
+### JSON Representation:
+
+```json
+{
+    "ptenant": "a60f3940-2ff4-4856-a191-17994f26c581",
+    "OneDriveDeviceId": "cd64acbe-bd10-48bd-ae04-4a443f5eaad6",
+    "User": {
+        "Name": "Kat Larson",
+        "Email": "KatL@contoso.com"
+    },
+    "DeviceName": "KAT-PC",
+    "SyncAppVersion": "23.199.0924.0001",
+    "SyncAppUpdateRing": 5,
+    "OSName": "Windows",
+    "OSVersion": "10.0.19042",
+    "SyncAppBuildType": "",
+    "KFMOptInWithWizardGPOEnabled": false,
+    "KFMSilentOptInGPOEnabled": false,
+    "KFMEnabledFolders": [
+        1,
+        2,
+        3
+    ],
+    "KFMEnabledFolderCount": 3,
+    "LastSyncedTimestampUTC": "2023-09-27T02:12:50Z",
+    "LastStatusReportedTimestampUTC": "2023-10-03T11:00:43Z",
+    "TotalErrorCount": 1,
+    "ErrorDetails": [
+        {
+            "OneDriveDeviceId": "cd64acbe-bd10-48bd-ae04-4a443f5eaad6",
+            "SyncErrorId": 53,
+            "TaskDialogId1": 0,
+            "TaskDialogId2": 256,
+            "ErrorTitle": "There isn't enough space on this computer to sync this file.",
+            "ErrorDescription": "Free up space by removing apps and files you don't need anymore. We'll automatically sync this file when there's enough space.",
+            "ErrorCount": 1
+        }
+    ],
+}
+```
+
+### Sample
+
+```json
+{"ptenant":"12345678-4d67-4c21-842a-abcea48840d5","OneDriveDeviceId":"848784b3-ba8e-425a-9e4e-a8635c061fe5","User":{"Name":"Robel Muris","Email":"Robe"lMu@contoso.com"},"DeviceName":"ROBE15432","SyncAppVersion":"23.201.0926.0001","SyncAppUpdateRing":4,"OSName":"Windows","OSVersion":"10.0.19042","SyncAppBuildType":"","KFMOptInWithWizardGPOEnabled":false,"KFMSilentOptInGPOEnabled":false,"KFMFolders":[3,1],"KFMFolderCount":2,"LastSyncedTimestampUTC":"2023-09-29T20:21:10Z","LastStatusReportedTimestamp":"2023-10-02T01:02:19Z","TotalErrorCount":0,"ErrorDetails":[],"SnapshotDate":"2023-11-02T00:00:00Z"}
```

Comment:
This e-mail for this user has an extra " for some reason.
Can you please fix to say "RobelMu@contoso.com".